src/frontends/onnx/frontend/src/core/tensor.hpp

```diff
@@ -260,8 +270,22 @@ class Tensor {
         std::shared_ptr<ov::op::v0::Constant> constant{nullptr};
         auto data = get_data<T>();
         auto data_size = data.size();
+        switch (type) {
```

Comment:
```suggestion
        ov::element::is_nibble_type(type) {
```
This helper is dev API `src/core/dev_api/openvino/core/type/element_iterator.hpp`